Source/WebKit/UIProcess/Cocoa/UIDelegate.h

```diff
@@ -68,6 +59,9 @@ class UIDelegate : public CanMakeWeakPtr<UIDelegate> {
     explicit UIDelegate(WKWebView *);
     ~UIDelegate();
 
+    void ref() const;
+    void deref() const;
```

Comment:
These are just empty declarations, right?

Should we use RefCountedAndCanMakeWeakPtr<UIDelegate> here, and offer a create() function, and so on?